src/components/containers.js

```diff
@@ -1,31 +1,46 @@
-/**
- * Components for laying out content within a consistently-styled container
- */
-
 import classnames from 'classnames';
 
 import { downcastRef } from '../util/typing';
 
+/**
+ * Components for laying out content within a consistently-styled container
+ */
+
 /**
  * @typedef {import('preact').ComponentChildren} Children
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLDivElement>} HTMLDivAttributes
+ * @typedef {import('preact').Ref<HTMLElement>} ElementRef
+ */
+
+/**
+ * @typedef CommonPresentationalProps
+ * @prop {Children} [children]
+ * @prop {string|string[]} [classes] - Optional extra CSS classes to append to
+ *   the component's default classes
+ * @prop {ElementRef} [containerRef] - Deprecated; use
```

Comment:
I assume the plan is to remove this in a semver-major release once downstream consumers have been migrated?

Reply:
Exactly! I was thinking a major release with the Tailwind conversion and removing this then. I was going to mention that in the next Changelog (i.e. "`containerRef` will be removed in vWhatever").